viz-lib/src/visualizations/addressable-table/Renderer.tsx

```diff
@@ -10,7 +10,7 @@ import { prepareColumns, initRows, filterRows, sortRows } from "./utils";
 import ColumnTypes from "./columns";
 
 import "./renderer.less";
-import NotEnoughData from '@/components/NotEnoughData';
+import NotEnoughData from '@/components/NotEnoughData'
```

Comment:
please add semicolon. 
change your default setting to semicolon and have other projects to get these errors